test/users.routes.test.js

```diff
@@ -566,35 +566,51 @@ describe('users.routes.test.js', () => {
     });
 
     it('fails if the user already has a password', done => {
```

Comment:
I don't know the exact intention, but that test isn't new. I just added code to have a donation for the user to make the test pass.